pkgs/gnolang/gnomod/gnomod.go

```diff
@@ -110,7 +114,7 @@ func GnoToGoMod(f File) (*File, error) {
 
 func isReplaced(module module.Version, repl []*modfile.Replace) (*module.Version, bool) {
 	for _, r := range repl {
-		if r.Old == module {
+		if (r.Old.Path == module.Path && r.Old.Version == "") || r.Old == module {
```

Comment:
```suggestion
		wasSameWithoutVersion := r.Old.Path == module.Path && r.Old.Version == ""
		if wasSameWithoutVersion || r.Old == module {
```

Not important for this one, but please, try to split complex conditions into something easier to understand.

Also, feel free to find a better name for the variable.

Reply:
Okay. I'll split complex conditions. but these conditions look simple and straightforward to me. What about adding an explanatory comment?


Also, I don't prefer mixed styling. If I were to do this, I will do it something like: 
```go
hasNoVersion := r.Old.Path == module.Path && r.Old.Version == ""
hasExactVersion := r.Old == module
if hasNoVersion || hasExactVersion {
...
}
```

Reply:
It was hard for me to understand the condition initially, maybe it will become harder when you come back to it in 6months :)

Adding a comment can help, but writing more obvious code that doesn't require a mention is even better.

I like your last version without mixing 👍 

FYI: I like using this method, especially in switch cases, when there are more than two variables, so we can group variable definitions in a `var (` section that will also set up indented code.

Reply:
Done: 4b735a99381984c6f44a11c6204d0ae88892ff02